QModManager/Messages.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+
+namespace QModManager
+{
+    public static partial class QModAPI
+    {
+        [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = false)]
+        public class MessageReceiverAttribute : Attribute
```

Comment:
Are nested classes necessary?  
Could this all be moved into a `QModManager.API` namespace instead?